recipes/asammdf/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set name = "asammdf" %}
+{% set version = "3.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set sha256 = "f7f644b76e4ffb59ad00c8e9377cf465fde85496688f15c7ca97be29a9855718" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+    - numpy >=1.13.1
+    - pandas
+    - numexpr
+    - matplotlib
+    - wheel
+
+test:
+  imports:
+    - asammdf
+    - test
```

Comment:
This shouldn't exist. Can you change this line `https://github.com/danielhrisca/asammdf/blob/master/setup.py#L75` to include `test`?

Reply:
should it be 
``packages=find_packages(exclude=['contrib', 'docs'], include=['test',]),``
or just without the _test_ exlude
``packages=find_packages(exclude=['contrib', 'docs']),`` ?

Reply:
It should be `packages=find_packages(exclude=['contrib', 'docs', 'test']),` instead of `packages=find_packages(exclude=['contrib', 'docs', 'tests']),`. Note `tests` vs `test`